lib/gollum/views/edit.rb

```diff
@@ -60,6 +60,10 @@ def is_edit_page
         true
       end
 
+      def critic_markup
```

Comment:
Isn't this superfluous given that `critic_markup` is defined in `layout.rb`? Or is the `Edit` view not a subclass of `Layout`?

Reply:
You are right. Will fix.